src/init.cpp

```diff
@@ -232,6 +194,8 @@ bool AppInit(int argc, char* argv[])
 
     bool fRet = false;
 
+    boost::shared_ptr<ThreadHandler> threads(new ThreadHandler);
```

Comment:
We're moving away from boost so you can change this to `std::shared_ptr` from `<memory>`.